packages/angular/pwa/BUILD.bazel

```diff
@@ -6,6 +6,8 @@
 load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
 load("//tools:defaults.bzl", "ts_library")
 load("//tools:ts_json_schema.bzl", "ts_json_schema")
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
```

Comment:
FYI: the `pkg_tar` rule from `bazel_tools` is deprecated and does not work on Windows. https://docs.bazel.build/versions/main/be/pkg.html

Not necessarily blocking for this PR as it probably is used elsewhere in the repo, but just a heads up.

Reply:
Noted. I'll look into changing those rules before the bazel built packages actually get used.